### ISSUE TITLE
Support global environment for CLI

### ISSUE THREAD

**Issue by moveurbody** [usebruno/bruno#3690]
### I have checked the following:

- [X] I've searched existing issues and found nothing related to my issue.

### Describe the feature you want to add

bruno-cli can support a global environment or specify an environment Json file.

e.g.`bru run --env-file {file_path}`

https://learning.postman.com/docs/collections/using-newman-cli/newman-options/#setup-options

### Mockups or Images of the feature

e.g.`bru run --env-file {file_path}`

https://learning.postman.com/docs/collections/using-newman-cli/newman-options/#setup-options

**Comment by sreelakshmi-bruno** [comment]
Hi @moveurbody, thanks for the suggestion! Adding it to our development roadmap.

**Comment by mesopotamiaMD** [comment]
Dear Bruno developlement, I need the mentioned functionality also, as:

`bru run --env-file ../../global_environment.json`

**global_environment.json** is already created via the UI, but BrunoCLI (bru run) doesn't support it. 

The global environments are needed because Collections can have subfolders, and REST Calls (scripts) can use created/returned variables of other calls globally. The variable exchange between two subfolders of a collection is possible via UI and not via BrunoCLI. 

In short, please implement/complete the functionality considering these points. 

Happy to get feedback soon. regards!

**Comment by mesopotamiaMD** [comment]
Dear @sreelakshmi-bruno, is there an update on this issue? Or do you have a workaround that can help too?

**Comment by anusree-bruno** [comment]
We are actively working on adding support for JSON environment files and this is expected to go in our next release.

**Comment by mesopotamiaMD** [comment]
Dear @anusree-bruno, thank you for the update. When do you expect the change will be available? If you have a pre-release version, I am glad to make some tests and give feedback to you. Best regards

**Comment by sreelakshmi-bruno** [comment]
Hi @mesopotamiaMD , supporting JSON files in the CLI has been delayed since we are working on some other crucial tasks. Because of this reason, I cannot give you an ETA on this but we'll update here when we pick it up.


> The global environments are needed because Collections can have subfolders, and REST Calls (scripts) can use created/returned variables of other calls globally. The variable exchange between two subfolders of a collection is possible via UI and not via BrunoCLI.

I agree, Bruno CLI currently lacks global environment support, which creates a gap between UI and CLI functionality for variable exchange between subfolders.

Current Solutions:

You can achieve cross-subfolder variable exchange using Bruno's scripting functions:

**Runtime Variables:**
```javascript
// In Request A (any subfolder)
bru.setVar('authToken', response.body.token);

// In Request B (different subfolder)  
const token = bru.getVar('authToken');
```

**Environment Variables:**
```javascript
// In Request A
bru.setEnvVar('userID', response.body.userId);

// In Request B
const userID = bru.getEnvVar('userID');
```

These work great for **single-session workflows** where you need variables to flow between requests across different subfolders.


However, global environments solve a different problem for **persistence**: Variables survive between sessions (unlike `setVar`/`setEnvVar`) and**pre-configuration**: Available immediately without running setup scripts.

Will the approach suggested above solve your use case? Also would you be open to a call to understand your use case better and get some feedback? Thanks!

**Comment by mesopotamiaMD** [comment]
Hi @sreelakshmi-bruno, Thank you for your feedback. I will check if your advice can help me work around my issue and give feedback on it.  At this time, the ticket https://github.com/usebruno/bruno/issues/4251 was created, that what you mentioned didn't work. 


In any case, I will be available to show you my project at any time. 

If the workaround works there are still some points, that a global_environment.json

- persistence topic as you mentioned
- transfers from another application like postman, doesn't work easily. 
- for each collection call we needed repeated entries, that can be done only in one global_environment.json 

Best regards

**Comment by sreelakshmi-bruno** [comment]
Hi @mesopotamiaMD , thanks for getting back. Please pick a time that works for you from [here](https://cal.com/sreelakshmi-jayarajan/product-discovery). Thanks!
